packages/map/src/index.jsx

```diff
@@ -15,4 +15,3 @@ export { default as Popup } from './popup';
 export * from './polygon/constants';
 export * from './circle/constants';
 export * from './pin_marker/constants';
-export * from './constants';
```

Comment:
All those constants have been exported by this package. Pretty sure it wasn't a conscious decision. If some app really depended on them then they have to help themselves - not willing to do a breaking release for that purpose